ivi-layermanagement-examples/EGLWLMockNavigation/src/Car.cpp

```diff
@@ -27,25 +27,28 @@ using std::endl;
 
 #include <GLES2/gl2.h>
 
-
 Car::Car(vec3f position, vec3f size, vec4f color, ShaderBase* shader)
 : m_position(position)
 , m_size(size)
 , m_color(color)
-, m_index(0, 1, 2)
+, m_index(0, 1, 2, 3)
 , m_pShader(shader)
 {
-	m_vertex[0].x = 0.0;
-	m_vertex[0].y = 0.0;
-	m_vertex[0].z = 0.0;
+    m_vertex[0].x = 0.0;
+    m_vertex[0].y = 0.0;
+    m_vertex[0].z = 0.0;
```

Comment:
no whitespace changes pls

Reply:
I've changed the indentation back to tabs, thank you!